build.sbt

```diff
@@ -23,17 +23,24 @@ ThisBuild / pomExtra := (
 
 publish / skip := true
 
+libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.8" % Test
+
 lazy val cross = crossProject(JSPlatform, JVMPlatform)
   .crossType(CrossType.Pure)
   .in(file("."))
   .settings(
-    name := "minitime"
+    name := "minitime",
+    libraryDependencies += "org.scalatest" %%% "scalatest" % "3.0.8" % Test
   )
   .jsSettings(
     libraryDependencies ++= Seq(
       "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC1"
     )
   )
 
-lazy val crossJVM = cross.jvm
-lazy val crossJS = cross.js
+scalafmtOnCompile in ThisBuild := true
+
+coverageMinimum := 70
+
+addCommandAlias("formatAll", "; scalafmt; test:scalafmt; scalafmtSbt")
+addCommandAlias("checkAllFormat", "; scalafmtCheck; test:scalafmtCheck; scalafmtSbtCheck")
```

Comment:
Can we use the `scalafmtCheckAll` command from scalafmt 2.0.0?

Reply:
Sure. In that time, scalafmt 2.0.0 only offered release candidate version, so I pick up the stable version.